cypress/e2e/tests/pages/manager/gke-cluster-provisioning.spec.ts

```diff
@@ -0,0 +1,113 @@
+import HomePagePo from '@/cypress/e2e/po/pages/home.po';
+import ClusterManagerListPagePo from '@/cypress/e2e/po/pages/cluster-manager/cluster-manager-list.po';
+import LoadingPo from '@/cypress/e2e/po/components/loading.po';
+import ClusterManagerCreateGKEPagePo from '@/cypress/e2e/po/edit/provisioning.cattle.io.cluster/create/cluster-create-gke.po';
+import LabeledSelectPo from '@/cypress/e2e/po/components/labeled-select.po';
+import { DEFAULT_GCP_ZONE } from '@/pkg/gke/util/gcp';
+
+// will only run this in jenkins pipeline where cloud credentials are stored
+describe('Deploy GKE cluster with default settings', { tags: ['@manager', '@adminUser', '@standardUser', '@jenkins'] }, () => {
+  const clusterList = new ClusterManagerListPagePo();
+  const loadingPo = new LoadingPo('.loading-indicator');
+
+  let cloudcredentialId = '';
+  const gkeDefaultZone = 'us-central1-c';
+  let gkeVersion = '';
+  // clusterId commented out as it will be used for the next tests of this suite, but not being used by the current test
+  // let clusterId = '';
+  let clusterDescription = '';
+
+  before(() => {
+    cy.login();
+    HomePagePo.goTo();
+
+    // clean up GKE cloud credentials
+    cy.getRancherResource('v3', 'cloudcredentials', null, null).then((resp: Cypress.Response<any>) => {
+      const body = resp.body;
+
+      if (body.pagination['total'] > 0) {
+        body.data.forEach((item: any) => {
+          if (item.googlecredentialConfig) {
+            const id = item.id;
+
+            cy.deleteRancherResource('v3', 'cloudcredentials', id);
```

Comment:
This will delete all gke credentials on the target rancher, without knowing it was there developers could lose them.

Can the test be updated to be agnostic of existing credentials?

Reply:
I could make this test agnostic to existing credentials, but the current cluster provisioning tests for the other cloud providers perform the same cleanup step. 
`cluster-provisioning-amazon-ec2-rke1.spec.ts` and `cluster-provisioning-amazon-ec2-rke2.spec.ts` perform it before the tests whereas `cluster-provisioning-azure-rke1.spec.ts` and `cluster-provisioning-azure-rke2.spec.ts` perform it after the tests. My suggestion is to keep the cleanup step.